lib/active_record_inherit_assoc.rb

```diff
@@ -60,7 +60,7 @@ def associate_records_to_owner(owner, records)
     end
   end
 
-  def scope
+  def build_scope
```

Comment:
does this need to be versioned?

Reply:
the signature has remained the same across all supported versions (i.e. between 5.0 and 6.1), as well as 7.0 and main at the time of writing.